.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: objective-c
```

Comment:
language is "objective-c" for both objective-c and swift. not sure why travis did it like this.